analysis/mode_queries/audience_overview.sql

```diff
@@ -0,0 +1,56 @@
+{#-
```

Comment:
@drewbanin had a look at this with me, and he also couldn't get it to work.
https://github.com/fishtown-analytics/dbt/issues/1152

Reply:
ok--maybe put this in a markdown file then? that way it won't attempt to be compiled.

Reply:
Going to keep it as the sql comment